runtime/doc/insert.txt

```diff
@@ -2005,8 +2005,15 @@ command in ex mode: >
 	one
 	two
 <
-NOTE: These commands cannot be used with |:global| or |:vglobal|.
-":append" and ":insert" don't work properly in between ":if" and
+When these commands are used with |:global| or |:vglobal| then the lines are
+obtained from the text following the command.  Separate lines with a NL
+escaped with a backslash: >
+	:global/abc/insert\
+	one line\
+	another line
+The final "." is not needed then.
```

Comment:
This only works in normal Ex mode. In cmdline mode or Vim Ex mode this doesn't work.

Reply:
I'll amend this then. Thanks